packages/terra-overlay/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Changelog
 Unreleased
 ----------
 ### Changed
+* Components updated to use `FormattedMessage` to interface with `react-intl's` `intl` context.
+
+3.24.0 - (July 30, 2019)
```

Comment:
We can remove this line and the `------------------` below it. We haven't cut a 3.24.0 release on this yet.